android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceInfoPackage.java

```diff
@@ -0,0 +1,56 @@
+package com.learnium.RNDeviceInfo;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.facebook.react.TurboReactPackage;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.module.model.ReactModuleInfo;
+import com.facebook.react.module.model.ReactModuleInfoProvider;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+
+public class RNDeviceInfoPackage extends TurboReactPackage {
```

Comment:
Does it work without `ReactModuleList` (see: https://github.com/software-mansion/react-native-screens/blob/5301d3f226a80ec7983e49209e8cee505d3a2ae7/android/src/main/java/com/swmansion/rnscreens/RNScreensPackage.kt#L12). I remember some issues when this code was not added, but maybe they've been solved already.